lizard_auth_client/urls.py

```diff
@@ -34,8 +34,8 @@ def check_settings():
                 'your settings. This URL is used to locate the SSO server.'
             )
     else:
-        if (not settings.SSO_SERVER_PUBLIC_URL or
-                not settings.SSO_SERVER_PRIVATE_URL):
+        if (not settings.SSO_SERVER_PUBLIC_URL
+                or not settings.SSO_SERVER_PRIVATE_URL):  # noqa: W503
```

Comment:
It is easier to remove the W503 check altogether. Lots of people think it is a wrong check anyway.
See https://black.readthedocs.io/en/stable/the_black_code_style.html#line-breaks-binary-operators

Something like this in the `setup.cfg` ought to help:

  [flake8]
  max-line-length = 88
  ignore = W503

Reply:
:) Done